pandas/core/indexes/datetimes.py

```diff
@@ -1747,7 +1747,33 @@ def freq(self, value):
     is_year_end = _field_accessor(
         'is_year_end',
         'is_year_end',
-        "Logical indicating if last day of year (defined by frequency)")
+        """
+        Return a boolean indicating whether the date is the last day of the
+        year.
```

Comment:
This should be a single line. Does

    Return a boolean array of whether the date is the year's last day.

fit on a single line?

Reply:
Boolean array is not correct in case of Series ... 

(but I agree that just "boolean" does not reflect is returns a boolean for each element)

Reply:
Perhaps just

    Return an indicator for whether each date is the last day of the year.

Reply:
Yes, that's better. Or maybe even shorter "Indicate whether ..." instead of "Return an indicator whether .." ?